app/src/main/java/helium314/keyboard/latin/inputlogic/InputLogic.java

```diff
@@ -794,6 +799,15 @@ private void handleFunctionalEvent(final Event event, final InputTransaction inp
         }
     }
 
+    private void handleSendIntentKey(int mKeyCode) {
+        var intentNumber = 1 + (-mKeyCode) - (-KeyCode.SEND_INTENT_ONE);
+
+        Intent intent = new Intent(mLatinIME.getString(R.string.action_send_intent));
+        intent.putExtra(mLatinIME.getString(R.string.extra_number), intentNumber);
+
+        mLatinIME.sendBroadcast(intent);
+        Log.i(TAG, "Sent broadcast for intent number: " + intentNumber);
+    }
```

Comment:
Could you move this code to a separate file? Right now it doesn't really matter, but I assume functionality for communicating with tasker and similar will (slowly) grow.
So I think having some `IntentUtils` or similar would be a good place to gather everything related. This could also hold the strings for action and extra, which should in my opinion not be in strings.